src/frontend/yarn.lock

```diff
@@ -0,0 +1,5827 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
need to test on 1) ubuntu 2) mac 3) windows and test with REST server running on LAN IP.

Reply:
also see, nothing is unexpected here and properly categorized as dev/ prod / test etc.